docs/contributing.md

```diff
@@ -4,9 +4,38 @@ title: Contribute to Stryker4s
 custom_edit_url: https://github.com/stryker-mutator/stryker4s/edit/master/docs/contributing.md
 sidebar_position: 3
 ---
-
-This is the contribution guide for Stryker4s. Great to have you here! Here are a few ways you can help to make this project better.
-
+## Developing on stryker4jvm:
+Currently, stryker4jvm is using 3 build tools for all its modules. Stryker4jvm-core is managed by maven,
```

Comment:
I think it would also help to have a picture (or ascii diagram) of the project layout as it's quite a bit more complex now